device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/transport/amqps/AmqpsIotHubConnection.java

```diff
@@ -1202,10 +1209,18 @@ private void openAsync() throws TransportException
         }
 
         this.reactor = createReactor();
+
+        String runnerUniqueIdentifier = this.isMultiplexing
+                ? "Multiplexed" + this.transportUniqueIdentifier
```

Comment:
Should there be a dash between "Multiplexed" and this.transportUniqueIdentifier?